### ISSUE TITLE
Roof leaks near chimney

### ISSUE THREAD

**Issue by nevinera** [nevinera/home#1]
There's plaster damage, and it apparently drains down the chimney box, because the wood at the bottom is in really bad shape.

**Comment by nevinera** [comment]
We had this "professionally" fixed, and now it leaks several times as much. The plaster is bulging, and the siding and one corner of the chimney box is rotting.

I'm going to get up there and figure this crap ou myself.

Why the hell did they remove the cricket?

**Comment by nevinera** [comment]
Looks like this is really about the edge of the roof, and not the shingles and box.
- aluminum flashing at edge so runoff will get to the gutter
- water guide to take roof water sideways away from the box
- replace the siding where it has started to rot

Longer term, we should add a gutter drop to that end, because water seems to pool up there.

**Comment by nevinera** [comment]
I have installed a water guide above the box, and added flashing at the edge of the roof.

The guide should channel water away from the chimney box, to prevent pooling and dribbling. Tithe flashing ought to cause the water to run off into the gutter instead of down the wall of the house.

Unfortunately, substantial damage was already done - I still need to:
- replace several rows of siding
- replace the 2x4 holding the inside edge of the chimney box against the wall
- remove and eventually replace the drywall above the fireplace.

The first two are blocking the third.

**Comment by nevinera** [comment]
Split remaining bits off into their own issues

**Comment by nevinera** [comment]
It's storming right now, and I went outside to take a look. Water is _sheeting_ down that wall of the house - my flashing has gotten the water to at least run into the gutter at the end, but the gutter is just overflowing and the spilloff hits the wall anyway. Added to that, the flashing only helps for the last 9 inches or so, which is where I assumed the problem to be, but the next few feet of gutter are exibiting the same issue - the water seems to largely miss the gutter on the way down.

I need to add a downspout for the gutter right there, to fix the overflow problem without having to reinstall the whole gutter. I also need to add flashing or shingles to make sure that the water all makes it into the gutter.

I can do a better job with that flashing too - I can git it the right length and bend it, so that the end is just overlapping the gutter, and then the water will flow properly. I might be able to do that with shingle instead - that would look better, so I should take a look at some shingles and see how easy it would be.

**Comment by nevinera** [comment]
The main issue appears to be that the gutter was bent down (probably by a contractor's foot), and pulled somewhat away from the house. Since the wood it was nailed into there is rotting, the giant nail that is supposed to hold it up can't get purchase.

I replaced a piece of siding lower down on the wall (poorly), but have not yet painted it. While I was replacing it, I found that both the plywood backing and much of the 2x4 framing is rotted beyond repair, and needs to be replaced, so I aborted my siding project - all of that siding will need to come off so I can replace the framing anyway.

**Comment by nevinera** [comment]
After that event, I repaired the gutter, by bending it back up and adding a board with shelf-brackets below it to hold it in place. The gutter ought to drain properly now, and it certainly will catch the runoff from the shingles correctly (the end of the shingles all overlap the gutter now).

I also spent a whole tube of caulk lining much of that section of wall with silicon - the particularly rotted siding boards near the top can still leak, but since that wood is all soft enough to chew already, there's not much point to trying to 'seal' it.
